README.rst

```diff
@@ -76,6 +76,11 @@ Installation
     ./venv.sh
     export PATH=$PWD/venv/bin:$PATH
 
+Usage with Docker
+-----------------
+
+If you want to use simp_le with Docker, have a look at https://github.com/zenhack/simp\_le/docker
```

Comment:
This is a dead link (and still will be once it's merged); paths within the repo are like `/tree/master/docker`.

Reply:
Will fix.